pkg/registration/spoke/bootstrapkubeconfigs.go

```diff
@@ -0,0 +1,75 @@
+package spoke
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	clusterv1client "open-cluster-management.io/api/client/cluster/clientset/versioned"
+
+	commonhelpers "open-cluster-management.io/ocm/pkg/common/helpers"
+)
+
+func selectBootstrapKubeConfigs(ctx context.Context, managedCluster string, bootstrapKubeConfigs []*rest.Config) (int, error) {
```

Comment:
add some comment. You are selecting based on order, and find the first available hub cluster.

Reply:
Comment added.